### PR TITLE
simplify build definition a little bit

### PR DESCRIPTION
**Pull request by SethTisue**
using my newfound sbt-fu, thx @dwijnand for the lesson

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -26,20 +31,6 @@ lazy val guiSettings: Seq[sbt.Def.Setting[_]] = Seq(
...
-lazy val root = Project(
-    id   = "root",
-    base = file("."),
-    // configure your native packaging settings here
-    settings = Seq(
-        maintainer           := "Adriaan Moors <adriaan@lightbend.com>",
-        packageDescription   := "Scala Bot",
-        packageSummary       := "Automates stuff on Github"),
-    // always run all commands on each sub project
-    aggregate = Seq(core, amazon, github, cli, jenkins, server)
```
Comment (dwijnand) [inline build.sbt]: from 6 back to aggregating all 8 sub-projects :)

**Comment by SethTisue** [comment]
the only thing I'm a bit unsure about here is dropping`.dependsOn(gui) // this does the actual aggregation` — not sure what that was supposed to mean/do

File: build.sbt (hunk 1 above)
Comment (SethTisue) [inline build.sbt]: the omission of `lightbend` here was just an oversight (mine), so it comes down to the status of `gui`

**Comment by SethTisue** [comment]
but, we haven't even been using the GUI :-\

**Comment by dwijnand** [comment]
Note that "gui" depends (transitively) on everything else. So depending on gui (currently) would making you effectively depend on every sub-project.

The comment is confusing.

Try asking early 2015 @adriaanm: https://github.com/scala/scabot/commit/3f46f0deba4f23c1444220e0d384e7cfc6f017a8

Note it used to depend on more before mid 2015 @adriaanm changed it: https://github.com/scala/scabot/pull/35/commits/eb7b978c46310c0d49d78b946228bd83a73bc4ab

**Pull request merged**